### ISSUE TITLE
Check pip version in setup.py

### ISSUE THREAD

**Issue by AbdealiJK** [coala-analyzer/coala#1913]
Pip changes a lot ...

We should check the pip version in our setup.py and tell users to upgrade pip.

@tushar-rishav got http://pastebin.com/0wMXLZmi
Whic is ugly and super confusing. And he was using v7.1.0

According to pip, `~=` should work on 6.0.0+ but it doesnt.

**Comment by AbdealiJK** [comment]
Hm, As @tushar-rishav pointed out - we cannot do this in setup.py.

Pip hasnt run the setup.py yet, so we checking the version there is useless.

We want to:
 - Figure out a way to upload some metadata on pypi to let pip know this.
 - Mention this in the readme as possible issues.

**Comment by AbdealiJK** [comment]
I cannot think of anyway to fix this. It seems like an error that is even before coala is un-zipped ...
Also, someone on gitter tried the same version and didn't get the error.
I tried reproducing it in a pyenv and was not able to either.
I'm not sure when/how this happens. But I guess even if we did, there's nothing we can do about it.

Closing this as pip does mention:
```
You are using pip version 7.1.0, however version 8.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```
on the top for every pip command if your pip version is old ...
